go/packages/dev-mode/agent/forwarder.go

```diff
@@ -143,6 +147,52 @@ func FindReqData(logs []LogItem) *LogItem {
 	return nil
 }
 
+func IsDefaultConsoleMessage(record interface{}) bool {
+	message := fmt.Sprintf("%v", record)
+	arr := strings.Split(message, "\t")
+	// Not enough items in the list
+	// The first item is not a date timestamp
+	// The second item is not a request id
+	if !hasInternalExtension || len(arr) < 4 || !IsValidDate(arr[0]) || !IsValidUUID(arr[1]) {
+		return false
+	}
+	// The third items is not a error or warning
+	// The third items is an error that does not contain an error object
+	isWarnOrError, logLevel := IsWarnOrError(arr[2])
+	if !isWarnOrError {
+		return false
+	} else if logLevel == "ERROR" && !HasErrorObject(arr[3]) {
+		return false
+	}
+	return true
+}
+
+func HasErrorObject(message string) bool {
+	foundErrorDeclaration := strings.Index(message, "Error: ")
+	foundTracePath := strings.Index(message, "/var/task")
+	if foundErrorDeclaration == -1 && foundTracePath == -1 {
+		return false
+	}
+	return true
+}
```

Comment:
This is where I am detecting if there is an error object in the error message string 👍

Reply:
@Danwakeem It looks good to me, it's probably best we can do. I have just two concerns:

1. This looks as Node.js specific, while dev mode should be runtime agnostic. Still I think it's fine to assume that at this point we just want to filter those error logs in case of Node.js runtime. Having that assumption, maybe it'll be worth to add an extra constraint which will ensure that this filter works only if we're in context of Node.js Lambda?
2. If there's a scenario where dev-mode works without SDK instrumentation (so we end with Lambda that has dev-mode layer attached but not SDK internal extension), then I think we need to ensure that this filtering is not applied if no SDK extension is present (as no events are sent to dev mode). Is that the case?

Reply:
Yeah I agree on both points 👍 both of these points should already be applied in the function that calls this function 😎